css-frameworks/vanilla vue ssr/src/ssr.js

```diff
@@ -0,0 +1,25 @@
+const renderer = __non_webpack_require__(
+  "vue-server-renderer"
+).createRenderer();
+import { app } from "./App";
+
+let html;
+
+renderer.renderToString(app).then(h => (html = h));
+
+export default function() {
+  return `
+    <!doctype html>
+    <html lang="en">
+      <head>
+        <meta charset="utf-8">
+        <title>CSS-Test</title>
+      </head>
+      <body>
+        <div id="root">
+          ${html}
+        </div>
+      </body>
+    </html>
+  `;
+}
```

Comment:
Not entirely sure but this should probably also work:

```suggestion
export default async function() {
  return `
    <!doctype html>
    <html lang="en">
      <head>
        <meta charset="utf-8">
        <title>CSS-Test</title>
      </head>
      <body>
        <div id="root">
          ${await renderer.renderToString(app)}
        </div>
      </body>
    </html>
  `;
}
```

Reply:
You're right about `async/await` and your suggestion works perfectly fine. Thank you!